libs/components/phone-field/testing/src/phonefield/phone-field-fixture.spec.ts

```diff
@@ -170,13 +171,26 @@ describe('PhoneField fixture', () => {
       'selectedCountryChange'
     );
 
-    // change the country
-    await phonefieldFixture.selectCountry(COUNTRY_AU.name);
+    if (COUNTRY_AU.name) {
+      // change the country
+      await phonefieldFixture.selectCountry(COUNTRY_AU.name);
+    }
+
+    const countryName: string | null =
+      await phonefieldFixture.getSelectedCountryName();
+
+    const countryIos2: string | null =
+      await phonefieldFixture.getSelectedCountryIso2();
```

Comment:
adding these so that code coverage is complete (these two functions were not being used ever)